tdrs-backend/tdpservice/users/api/login.py

```diff
@@ -160,9 +160,9 @@ def handle_user(self, request, id_token, decoded_token_data):
                 user.save()
 
             if user.deactivated:
-                self.login_user(request, user, "Inactive User Found")
+                login_msg = "Inactive User Found"
             else:
-                self.login_user(request, user, "User Found")
+                login_msg = "User Found"
```

Comment:
I would move this `login_msg = "User Found"` higher up in the scope and remove this else block